tests/virt_autotest/libvirt_isolated_virtual_network.pm

```diff
@@ -62,10 +62,13 @@ sub run_test {
 
         assert_script_run("virsh attach-interface $guest network vnet_isolated --model $model --mac $mac --live $affecter", 60);
 
-        my $net = (get_var('XEN') || check_var('SYSTEM_ROLE', 'xen') || check_var('HOST_HYPERVISOR', 'xen') && is_sle('=11-sp4')) ? 'netfront' : 'vnet_isolated';
+        my $net = is_sle('=11-sp4') ? 'br123' : 'vnet_isolated';
         test_network_interface($guest, mac => $mac, gate => $gate, isolated => 1, net => $net);
 
         assert_script_run("virsh detach-interface $guest --mac $mac $exclusive");
+
+        #cleanup file /etc/hosts after finished virtual network test
+        script_run "sed -i '/$guest/d' /etc/hosts";
```

Comment:
SR-IOV test need this kind of cleanup stuff as well. Can we have a common function in base or utils? including removing /root/.ssh/known_hosts

Reply:
/root/.ssh/known_hosts is not necessary as it is just for local test.